src/tic_tac_toe/referee.clj

```diff
@@ -11,3 +11,36 @@
 (defn validate-move
   [player move]
   (convert-to-spot move))
+
+(defn draw?
+  [board]
+  (board/full? board))
+
+(defn matching-pattern?
+  [token pattern]
+  (loop [token token
+         pattern pattern]
+    (match pattern
+      [token] true
+      [" " & r] false
+      [(_ :guard #(not= token %)) & r] false
+      [token & r] (recur token (vec (rest pattern))))))
+
+(defn winner?
+  [board]
+  (let [patterns (board/give-patterns board)]
+    (loop [patterns patterns]
```

Comment:
You could get by without adding the `let`, since `loop` gives you an implicit `let`. Your call though 😄

Reply:
[Refactored this area a bit.](https://github.com/saralein/tic-tac-toe_clojure/pull/6/files#diff-f014d0e7db79dee6fc42af72f74d5325R54).  It no longer loops, but I also make sure to not use let where it isn't needed.